app/views/trips/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<p class="title"><%= t(".title") %></p>
+<table>
+  <%= render "trips_table_header" %>
+  <%= render "trip" %>
+  <td class="title">Still to allocate</td>
+  <%= render "people_to_allocate" %>
+  <td>
+    <%= link_to t("tripfamilies.new.associate_families"),
+    new_trip_tripfamily_path(trip),
+    class: "list-button" %>
+  </td>
+</table>
+
+<table>
+  <p class="title"><%= t(".allocated_families") %></p>
+  <% @tripfamilies.each do |tripfamily| %>
+  <tr>
+    <td>
+    <% if tripfamily.trip_id = trip.id %>
```

Comment:
No need for the `if`, because you can use `trip_family.family.last_name`
And should start considering using a presenter here